sys-process/bashtop/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<use>
+		<flag name="sysstat">System Performance tool for Linux</flag>
+	</use>
+</pkgmetadata>
```

Comment:
Add yourself as proxied maintainer in metadata.xml.  Also add me as gentoo manitainer.

Reply:
I have added the maintainers to metadata.xml

Reply:
Usually put yourself first, then the Gentoo maintainer, then proxy-maint project if needed (but given you guys have a relationship already, may not be needed).

This is so you get assigned the bugs and so on.

Reply:
The commit message is not correctly formed. otherwise is ready to be merged.

the commit message must be of the form:
sys-process/bashtop: New package...

anything else relevant to the commit...
Signed-off-by:  Foobar...


Fix this and I can merge :)

Reply:
I have committed as per your suggestion